l10n_it_delivery_note/wizard/delivery_note_base.py

```diff
@@ -15,6 +15,13 @@ def _default_stock_pickings(self):
 
         return self.env["stock.picking"].browse(active_ids)
 
+    def _domain_type_id(self):
+        return [
+            "|",
+            ("company_id", "=", False),
+            ("company_id", "=", self.env.company.id),
```

Comment:
```suggestion
            ("company_id", "in", [False, self.env.company.id]),
```
non ci sono comunque problemi a lasciare com'è.